wazo_confd/plugins/user/resource.py

```diff
@@ -92,11 +93,9 @@ def _post(self, body):
             self._user_line_item_resource.put(user_dict['uuid'], line['id'])
             user_dict['lines'].append(line)
 
-        # FIX: create(...) takes a user dict containing an 'email_address' key, not an 'email' key
-        if user_dict.get('username'):
-            fixed_user_dict = user_dict.copy()
-            fixed_user_dict['email_address'] = fixed_user_dict['email']
-            self._wazo_user_service.create(fixed_user_dict)
+        if auth:
```

Comment:
I didn't read the PR (or previous) and context, but according old spec and user import, the wazo-auth user should always be created, it is a part of the contract
With this logic, it will be a headache to build UI to fix confd user with auth user

Reply:
If wazo-auth is unreachable a 503 is returned

Reply:
The auth user is only created when the `auth` section is specified in the body. If someone uses a the body from a previous version the auth user is not created so the behavior stays the same as before